test-packages/static-app/package.json

```diff
@@ -56,5 +56,8 @@
   },
   "engines": {
     "node": "8.* || >= 10.*"
+  },
+  "resolutions": {
+    "ember-auto-import": "stefanpenner/ember-auto-import#lazily-read-babel-config"
```

Comment:
I suspect this resolution isn't working properly on CI, but appears to be fine locally. With this change, this CI work lands and we are good.